openedx/core/djangoapps/kafka_consumer/docs/0001_kafka_consumer_command.rst

```diff
@@ -0,0 +1,39 @@
+Managing Kafka Consumers
+--------------
+
+Status
+======
+
+In Progress
+
+Context
+=======
+As outlined in the upcoming OEP-52, we have elected to go with Apache Kafka as
+our event bus implementation. The standard pattern for consuming events with
+Kafka is to poll in a loop and process messages as they come in. According to
+the Confluent team, it is a best practice to limit each consumer to a single
+topic.::
+    
+    consumer.subscribe(["topic"])
+    while True:
+        message = consumer.poll()
+        ## process message
+
+This ``while True`` loop means whatever is running this consumer will run infinitely and block whatever thread runs it from doing anything else. Thus, we can’t just run this code as part of our regular Django web server. 
+
+Decision
+========
+We will use Kubernetes to manage containers whose sole purpose is to run a management command, which in turn will run a polling loop against the specified topic.
+
+Options Considered
+==================
+
+#. Use celery tasks to consume a set number of messages at a time use our existing celery structure
+    * This would likely require us to use the celery-beat library to run the celery tasks on a constant basis. Celery Beat has known            performance issues and would likely not scale well as the number of topics increased.
```

Comment:
1. Consider using brief sub-headers for each of these options. [Optional]
2. Additionally, any cron-like polling has other design flaws due to the potential for polling to be misaligned with the time required to process any new events.